src/Main.ts

```diff
@@ -867,12 +921,17 @@ export class Main {
             // Check to see if we have a room for this channel already.
             const existing = this.rooms.getBySlackChannelId(openResponse.channel.id);
             if (existing) {
-                await this.datastore.deleteRoom(existing.InboundId);
                 await intent.sendEvent(roomId, "m.room.message", {
                     body: "You already have a conversation open with this person, leaving that room and reattaching here.",
                     msgtype: "m.notice",
                 });
-                await intent.leave(existing.MatrixRoomId);
+                try {
```

Comment:
It certainly does: since the bridge sets a custom name/avatar on Slack DM rooms, that name/avatar will persist even after the Slack ghost leaves the room -- meaning it will continue to look like an active DM in the room list despite being empty!  Removing the name/avatar allows clients to calculate the room name & choose to fall back to a title of "Empty room" or whatever.